### PR TITLE
lutaml2dotpng script, small fixes

### PR DESCRIPTION
**Pull request by w00lf**
metanorma/basicdoc-models#17 Script to convert wsd files(plantuml) in to lutaml, fixes lutaml2dotpng script, render data_types/primitives graphviz, support for windows line endings - \r\n

### CODE REVIEW

File: lutaml-uml.gemspec [hunk 1]
```diff
@@ -29,10 +29,11 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "parslet", "~> 1.7.1"
   spec.add_runtime_dependency "ruby-graphviz", "~> 1.2"
   spec.add_runtime_dependency "thor", "~> 1.0"
+  spec.add_runtime_dependency "expressir", "~> 0.2.0"
```
Comment (ronaldtse) [inline lutaml-uml.gemspec:32]: Why is lutaml-uml requiring expressir?

Comment (w00lf) [reply lutaml-uml.gemspec]: Removed

**Comment by ronaldtse** [comment]
@w00lf can you:

1. Move lutaml2dotpng code into `lutaml` command:
`lutaml --via=dot my-lutaml-file.lml`

2. Use the `ruby-graphviz` gem instead of calling `dot` directly?
https://github.com/glejeune/Ruby-Graphviz#synopsis
```ruby
    require 'ruby-graphviz'

    # In this example, hello.dot is :
    #   digraph G {Hello->World;}

    GraphViz.parse( "hello.dot", :path => "/usr/local/bin" ) { |g|
      g.get_node("Hello") { |n|
        n[:label] = "Bonjour"
      }
      g.get_node("World") { |n|
        n[:label] = "Le Monde"
      }
    }.output(:png => "sample.png")
```

Thanks!

**Comment by w00lf** [comment]
> @w00lf can you:
> 
> 1. Move lutaml2dotpng code into `lutaml` command:
> 
> ```shell
> lutaml -t dot -o my-lutaml-file.dot my-lutaml-file.lml
> dot -T png my-lutaml-file.dot > my-lutaml-file.png
> 
> # or just
> lutaml -t my-lutaml-file.png --via=dot my-lutaml-file.lml
> ```
> 
> 1. Use the `ruby-graphviz` gem instead of calling `dot` directly?
>    https://github.com/glejeune/Ruby-Graphviz#synopsis
> 
> ```ruby
>     require 'ruby-graphviz'
> 
>     # In this example, hello.dot is :
>     #   digraph G {Hello->World;}
> 
>     GraphViz.parse( "hello.dot", :path => "/usr/local/bin" ) { |g|
>       g.get_node("Hello") { |n|
>         n[:label] = "Bonjour"
>       }
>       g.get_node("World") { |n|
>         n[:label] = "Le Monde"
>       }
>     }.output(:png => "sample.png")
> ```
> 
> Thanks!

Removed this script completely, we can use lutaml-uml script instead

**Pull request merged**